src/index.js

```diff
@@ -236,6 +236,8 @@ class Migrate extends Command {
         continue;
       }
 
+      await codeshift[sdk.language].install();
```

Comment:
I don't think this is necessary anymore now that we're using install in the `codeshift.run` util.